tests/pgsql/pgsql-bug-6983-ips/suricata.yaml

```diff
@@ -16,3 +16,6 @@ app-layer:
   protocols:
     pgsql:
       enabled: yes
+
+detect:
+  guess-applayer-tx: true
```

Comment:
should there also be a test.yaml update with this test?

Reply:
No, why should there be ?

Reply:
to include that the tx was guessed? (I guess?)

Reply:
I do not think it is important... Is it ?

Reply:
what is the point of the change if there is no test update with it?

Reply:
TL;DR Otherwise this test pgsql-bug-6983-ips fails with the Suricata PR.

I think you are referring to Suricata change here : https://github.com/OISF/suricata/pull/12197/files#diff-ee484ae4b77e59eb8b6b50f628c84ab626bd6178a6d43e1219dbc9619be7e027L816

Over simplifying it : instead of using `(alert_flags & PACKET_ALERT_FLAG_STREAM_MATCH)` we now use `de_ctx->force_applayer`

pgsql-bug-6983-ips resorted to `(alert_flags & PACKET_ALERT_FLAG_STREAM_MATCH)` to have pgsql metadata in alerts.
With the Suricata PR, we now need `de_ctx->force_applayer` meaning `detect.guess-applayer-tx: true` in yaml

That comes from how I understood your comment https://github.com/OISF/suricata/pull/12166#discussion_r1861730832

> We've decided it's worse to log the wrong one, than to log none.

Beyond pgsql-bug-6983-ips test, this SV PR has test bug-7199 to show the new benefits of the new behavior.